pkg/credentials/chain.go

```diff
@@ -58,28 +56,30 @@ func NewChainCredentials(providers []Provider) *Credentials {
 	})
 }
 
-// Retrieve returns the credentials value or error if no provider returned
-// without error.
+// Retrieve returns the credentials value, or no anonymous credentials if
```

Comment:
This comment is quite confusing - s/no anonymous credentials/the anonymous credential value/ perhaps.

Reply:
fixed